components/Pagination.js

```diff
@@ -79,9 +78,8 @@ const Pagination = ({ route, limit, offset, total, scrollToTopOnChange, isDisabl
       </Container>
       {currentPage < totalPages && (
         <Link
-          route={route || router.route.slice(1)}
+          href={{ pathname: route || router.route.slice(1), query: { ...router.query, offset: offset + limit } }}
```

Comment:
@Betree : For this one, actually I saw this initially but missed writing it in my initial post. Now the problem here is that the query object returned by the router has the full query details including the slug. 

> There are a bunch of `href` still not including a leading `/`. The following regex helps to find them (tried with VSCode):
> 
> ```
> # For links
> href=\{[`'][^/#(HTTP))]
> 
> # For pushRoute
> router.push\([`'][^/#(HTTP))]
> ```

@Betree : Sorry about this, it seems I missed some due to not using the full search in Intellij but rather a short-cut method. But now I've corrected all of them. 😄

Reply:
@Betree : I saw this when I was testing initially but somehow missed writing it down. I was thinking whether this should be fixed or not as I don't see it as a problem? 🤔 Now in the NextJS router the query includes the slug whereas previously it didn't and was part of the pathname. If we are to fix it since we are using the router object here, I was thinking it would perhaps be to change the passed route name to include the slug and not using the `router.query` within the `query` parameter. Anyways let me know your thoughts.

Reply:
We do want this route to be properly formatted. We can eventually solve it in a separate issue, but it would be great if we can do it here. It's probably the same solution that we need for https://github.com/opencollective/opencollective-frontend/pull/5795.

My understanding of the problem is not good enough to give recommendations, but I can look at it tomorrow.

Reply:
@Betree : Not to worry, I was under the impression that's a valid route but I think we are opting for routes without query parameter correct? Anyways, I've pushed my change (https://github.com/opencollective/opencollective-frontend/pull/5855/commits/889af8f694ac7ceb47cab7de4ec83d9e4b4c00f6) for that and that should take care of it. Let me know if you see any issues. 😄